### ISSUE TITLE
Javascript not working on using forward and backward buttons

### ISSUE THREAD

**Issue by vikas32567** [chezuba/chezuba-prototype#9]
Image source not set to default when redirected from another page using FORWARD or BACKWARD buttons

**Comment by mkvalluri** [comment]
Did you commit the code related to this?

**Comment by vikas32567** [comment]
Yeah... Could you see the default image assigning at first but if you go to other page and come back(using ```BACK``` button) That image is not loading.
